pkg/pillar/types/zedroutertypes.go

```diff
@@ -3169,6 +3169,24 @@ func GetIPAddrCountOnSubnet(subnet net.IPNet) int {
 	return 0
 }
 
+// GetIPAddrCountInRange returns count of addresses between two of them
+func GetIPAddrCountInRange(ip1, ip2 net.IP) int {
+	//TBD:XXX, IPv6 handling
+	ip1v4 := ip1.To4()
+	ip2v4 := ip2.To4()
+	if ip1v4 == nil || ip2v4 == nil {
+		return 0
+	}
+	ip1Int := uint32(ip1v4[0])<<24 + uint32(ip1v4[1])<<16 +
+		uint32(ip1v4[2])<<8 + uint32(ip1v4[3])
+	ip2Int := uint32(ip2v4[0])<<24 + uint32(ip2v4[1])<<16 +
+		uint32(ip2v4[2])<<8 + uint32(ip2v4[3])
```

Comment:
At least this is done in a better way in an existing function.

Reply:
I can see exactly the same in AddToIP. Moved to the library function.